manifests/prometheus/alerts.d/aiven-cost.yml

```diff
@@ -7,8 +7,7 @@
     name: AivenEstimatedCostHigh
     rules:
       - alert: AivenEstimatedCostHigh
-        expr: paas_aiven_estimated_cost_pounds > 6500
-        for: 1h
+        expr: delta(paas_aiven_estimated_cost_pounds[24h]) * 30 > 20000
```

Comment:
If we want to use the correct number of months:
```
delta(paas_aiven_estimated_cost_pounds[24h]) * days_in_month(timestamp(paas_aiven_estimated_cost_pounds)) > 0
```

Or simply we can change it to 31 so it always considers the highest possible number of days.

Also should we add some smoothing, so the graph is not that spikey?

Reply:
Good suggestions - but I think it's actually nicer to assume all months are 30 days so the rate is constant. Smoothing would be good too, but I don't think it's worth the effort to implement.